Gemfile

```diff
@@ -10,14 +10,14 @@ gem "aws-sdk-sqs", "~> 1.80"
 gem "bootsnap", "~> 1.18"
 gem "clearance", "~> 2.7"
 gem "dalli", "~> 3.2"
-gem "datadog", "~> 2.1", require: "datadog/auto_instrument"
+gem "datadog", "~> 2.1"
 gem "dogstatsd-ruby", "~> 5.5"
 gem "google-protobuf", "~> 4.27"
 gem "faraday", "~> 2.9"
 gem "faraday-retry", "~> 2.2"
 gem "good_job", "~> 3.29"
 gem "gravtastic", "~> 3.2"
-gem "honeybadger", "~> 5.5.1" # see https://github.com/rubygems/rubygems.org/pull/4598
```

Comment:
why this change?

Reply:
requiring the gem in the gemfile was too early to configure eg logging. moving the require to the initializer means we can suppress the warning in development about exceptions not being reported to the web service.